jpa/eclipselink.jpa.testapps/jpa.test.ddlgeneration/src/test/java/org/eclipse/persistence/testing/tests/jpa/ddlgeneration/FractionalSecondsPrecisionTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ * Copyright (c) 2025 IBM Corporation. All rights reserved.
```

Comment:
Again.

Reply:
The same here - based on IBM code from https://github.com/eclipse-ee4j/eclipselink/issues/2451.

Reply:
Rewritten to contain only my own code.